backend/typescript/yarn.lock

```diff
@@ -6194,4 +6194,4 @@ yn@3.1.1:
 yocto-queue@^0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/yocto-queue/-/yocto-queue-0.1.0.tgz#0294eb3dee05028d31ee1a5fa2c556a6aaf10a1b"
-  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
+  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
```

Comment:
(minor) Accidental commit? :shushing_face:

Reply:
I am not sure why that's being registered as a change. I put it through diff checker and it says the 2 files are identical. If there is a change, it's probably in the white spacing or something. The contents should be the same

Reply:
It looks like you removed a newline char maybe? Try restoring this file from development and committing